src/jackdaw/serdes/resolver.clj

```diff
@@ -39,6 +39,8 @@
   Options:
   schema-registry-url - The URL for the schema registry
   type-registry - A mapping per jackdaw.serdes.avro/+base-schema-type-registry+>
+  read-only - Specifies that you will not be using the resulting serializer,
+              and does not require a schema or schema-filename
```

Comment:
What will happen if you try to write with a read-only avro serdes?
Would be nice to get a clean error message, as debugging avro error is a black art at the best of times!

Reply:
I've added an informative message, and a test of that message.